src/commcare_cloud/ansible/deploy_hq.yml

```diff
@@ -2,6 +2,12 @@
 # similarly named deploy_commcarehq.yml playbook is used to configure machines
 # to run commcare-hq, and is expected to be run less frequently than this one.
 
+- name: Check hosts Ubuntu version
+  hosts:
+    - all
+  tasks:
+    - import_role: {name: upgrade_warning}
```

Comment:
I think we should make it possible to disable this check. One way to do that would be to use `include_role` here and add a `when: ...` condition that checks for the presence of a flag (file) on the control machine or something like that.

Reply:
I added the tag `upgrade_warning` to this role when called by the playbook, and instructions in the prompt message to add `--skip-ansible "upgrade_warning"` to the command used in order to skip the warning.
Does this solution work?

Reply:
The purpose of this override is to allow third parties who know what they are doing to disable the warning. What do you think of providing a more permanent way to disable it, such as an environment variable? Something like adding `export COMMCARE_CLOUD_IGNORE_UNSUPPORTED_OS=1` to bash profile, for example. Forcing them to always add an extra argument to the deploy command will probably get annoying.

Reply:
I think we may actually want it to be a bit annoying, at least until the deadline.
If the option is used, we know it will be visible, reminding the user of the potential risk.
After the deadline, we can definitely add an environment variable or similar solution to avoid using an explicit option, but in the meantime I think the purpose of this is also to be a bit bugging.

Reply:
I see no need to be annoying to people who know what they are doing. But I'm not going to block this PR over that disagreement.